pipeline/splitters.go

```diff
@@ -23,9 +23,14 @@ import (
 	"crypto/subtle"
 	"errors"
 	"fmt"
-	"github.com/mozilla-services/heka/message"
 	"hash"
 	"regexp"
+
+	"github.com/mozilla-services/heka/message"
+)
+
+const (
+	MULTILINE_SPLITTER_MAX_LINES = 99
```

Comment:
Seems like this should be a config option on the plugin rather than a hard-coded constant.